cmd/stunstamp/stunstamp.go

```diff
@@ -509,20 +509,20 @@ func (r *remoteWriteClient) write(ctx context.Context, ts []prompb.TimeSeries) e
 
 func remoteWriteTimeSeries(client *remoteWriteClient, tsCh chan []prompb.TimeSeries) {
 	bo := backoff.NewBackoff("remote-write", log.Printf, time.Second*30)
+	var writeErr error
```

Comment:
I'm curious why you pulled this out of the scope? It doesn't seem to be used anywhere else. (The compiler is reasonably smart about automatics if that's the concern)

Reply:
Added:
```
	// writeErr may contribute to bo's backoff schedule across tsCh read ops,
	// i.e. if an unrecoverable error occurs for client.write(ctx, A), that
	// should be accounted against bo prior to attempting to
	// client.write(ctx, B).
```

Reply:
SGTM, thanks for the comment!